### ISSUE TITLE
Planet rescan should not block ScoreReport

### ISSUE THREAD

**Issue by kerzyte** [SteamDatabase/SalienCheat#145]
If a Planet re-scan takes longer than the time frame for submitting a ScoreReport, you will be given an E27 once the planet re-scan is able to continue.

**Comment by xPaw** [comment]
I have only seen that happen when a new planet appears, so the API dies regardless.

**Comment by kerzyte** [comment]
Fair point, I was under the assumption that it was only the GetPlanet* calls that were dying.
